treeherder/intermittents_commenter/commenter.py

```diff
@@ -314,28 +334,41 @@ def get_bug_stats(self, startday, endday):
 
         for bug in bugs:
             platform = bug["job__machine_platform__platform"]
+            test_variant = bug["job__signature__job_type_name"].rsplit("-", 1)[0]
```

Comment:
this is a good shortcut, but will not work long term, here are a few cases to consider:
1) we have job_type_name that is a set of tests run in parallel, so these end with numbers (i.e. mochitest-plain-1)
2) some variants have multiple `-` in them: https://searchfox.org/mozilla-central/search?q=suffix&path=variants.yml&case=false&regexp=false
3) sometimes we don't have a variant at all;  

of these, 1 and 2 need to be solved, we can hack around #3 as that is normal, but in the scope of parsing, it could be agreed as an "edge case".

as this is python code, we could copy it from another source (a [phab request](https://d2mfgivbiy2fiw.cloudfront.net/file/data/twwlmagw2p5pt2a2jxgf/PHID-FILE-cwnpsk575nlvf3abw7e6/D230177.1732828095.diff) parsing task names to find variants, search for `found_variants`).  When that code lands, I have some refactoring I want to do so eventually this will become an easier library to reuse.
 * in the case of the example code, found_variants is a list, you can probably join them with a '+' or a '-' for the purposes of the table header.
 
 
we should update the sample test data in order to have more modern and useful task names. 

The rest of the changes look good!